share/data-modeling/domain-driven-design/pattern1/README.md

```diff
@@ -0,0 +1,183 @@
+# Pattern 1: Single Domain, Single Bounded Context, and a Single Uni
+
+
+# Scenario
+
+The customer Relationship Management(CRM) Team wants to build a single source of truth for customer account data which includes first name, last name, customerId, personal details, contact details, and address. The CRM team will frequently query customer data using customerId. 
+
+
+# Modeling
+
+Since CRM is an independent function we will assume it is a domain on its own. Inside CRM, we will have multiple sub-domains; customer account is one such sub-domain. You can model a customer account as a Bounded Context.  Since CRM will perform frequent lookup, search, and filter on customer records by customerId, you will index customer accounts on customerId property. 
+
+
+# Create a Uni
+
+1. Login to Vendia Share [here](https://share.vendia.net/login). 
+
+Don’t have a login, you can sign up for one [here](https://share.vendia.net/). 
+
+2. Create Uni
+
+Click the 'Create Universal Application' button on the top-right. Next, use the ‘Create your own option. 
+
+3. Fill out details in the Uni Creation Wizard. 
+
+1. **Step 1:** Give your Uni a name. Since this Uni will be owned and operated by the CRM team. I will give it the name ‘CRM Uni’. Click ‘Next’
```

Comment:
We should recommend:

1. They prefix their Uni name with `test-` so its name can be reused
2. They pick a unique Uni name since the `vendia.net` namespace is shared by all Starter and Individual Tier users, which will cause naming collisions

Reply:
I will also add a suggestion to delete Uni at the end of each pattern as free tier will have license to create a  single Uni